opentelemetry-sdk/setup.cfg

```diff
@@ -42,6 +42,7 @@ zip_safe = False
 include_package_data = True
 install_requires =
     opentelemetry-api == 1.0.1.dev0
+    opentelemetry-semantic-conventions == 1.0.1.dev0
```

Comment:
i wouldnt expect this package to be marked as stable as the spec is still experimental

Reply:
Then does it still make sense to use this unstable package in sdk to derive the resource attributes?

Reply:
I think that's fine, it just means that the SDK will need to be updated as the conventions package is updated. @lzchen any thoughts?

Reply:
Should be fine.